Week3/akif-week3.js

```diff
@@ -0,0 +1,158 @@
+//   STRINGS! --------------
+let myString = 'hello,this,is,a,difficult,to,read,sentence';
+console.log(myString);
+console.log(myString.length);
+myString = myString.replace(/[,]/g, ' ');
+console.log(myString);
+
+//   ARRAYS! -------------------------------------------------
+let favoriteAnimals = ['blowfish', 'capricorn', 'giraffe'];
+favoriteAnimals.push('turtle');
+console.log(favoriteAnimals);
+favoriteAnimals.splice(1, 0, 'meerkat');
+console.log("i added a new value 'meerkat' to the array");
+console.log(favoriteAnimals);
+console.log('The array has a length of: ' + favoriteAnimals.length);
+favoriteAnimals.splice(3, 1);
+console.log(favoriteAnimals);
+console.log('The item you are looking for is at index: ' + favoriteAnimals.indexOf('meerkat'));
+favoriteAnimals.splice(favoriteAnimals.indexOf('meerkat'), 1);
+
+//  MORE JAVASCRIPT ! ------------------------------------------------
+// -----1111111111111111111-----------
+function getSum(a, b, c) {
+  return a + b + c;
```

Comment:
A pure function this is since it is not updating any variable outside its ownership 🙂